### ISSUE TITLE
Add the AllContributors app

### ISSUE THREAD

**Issue by NicMcPhee** [UMM-CSci-3601/3601-iteration-template#596]
The [AllContributors](https://github.com/apps/allcontributors) app looks like it might be a neat way to ensure that everyone who has contributed to a project gets "automagically" recognized at the bottom of the README. Not a vital thing, but it would make sense as a thing to include if it's not hard.

If it works, it should be added to the labs as well.

**Comment by NicMcPhee** [comment]
@all-contributors please add @kklamberty for ally, code, content, doc, design, ideas, mentoring, projectManagement, question, test, tutorial

**Comment by NicMcPhee** [comment]
I added each of @floogulinc, @helloworld12321, and @kklamberty, and it definitely works.

Based on this experiment, I don’t think I’m going to include this in any of the labs. We _might_ want to include it in the iterations (I think it could provide a nice visual record of people’s contributions), but I think it’s going to be more work and confusion than it’s worth in the labs.

One thing that definitely was an issue is that this clearly isn’t meant to be used “in parallel”. By doing contributions for each of these three people at the same time, I ended up with some annoying merge conflicts that weren’t entirely trivial to resolve. It also doesn’t update the badge properly when you do them in parallel.

My guess is that students will just get bogged down with this, especially early in the semester.
